src/containers/SpecialTextBoxContainer.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from "react-redux";
+import setSpecialText from "../actions";
```

Comment:
Go back and review the redux address book for importing actions.
How do you import things that aren't the default export